packages/itwinui-react/src/core/utils/components/WithCSSTransition.tsx

```diff
@@ -24,21 +23,21 @@ export const WithCSSTransition = (
       in={visible}
       timeout={200}
       unmountOnExit={true}
-      onEnter={(node) => {
+      onEnter={(node: HTMLElement) => {
         node.style[`min${dimensionCamelCase}`] = 'initial';
         node.style[dimension] = '0px';
       }}
-      onEntering={(node) => {
+      onEntering={(node: HTMLElement) => {
         node.style[dimension] = `${expandedSize.current}px`;
       }}
-      onEntered={(node) => {
+      onEntered={(node: HTMLElement) => {
         node.style[`min${dimensionCamelCase}`] = '';
         node.style[dimension] = '';
       }}
-      onExit={(node) => {
+      onExit={(node: HTMLElement) => {
         node.style[dimension] = `${expandedSize.current}px`;
       }}
-      onExiting={(node) => {
+      onExiting={(node: HTMLElement) => {
```

Comment:
nit: Types for `node` in `onExit` and `onExisting` does not seem to be needed. But type does not seem to be inferred in the other three callbacks. So, if its needed in three places, might also be alright to define the type in the remaining two places too, like you have it.

Reply:
yeah lets leave it this way for consistency i guess